### PR TITLE
Fix rendering app authors with homepage or email

### PR DESCRIPTION
**Pull request by ChristophWurst**
When the first author of an app doesn't have homepage/email attributes
set in info.xml then any further author was rendered as `[object
Object]` due to the complex XML node serialized to a JavaScript string.
The logic was fixed so that it converts any of the authors to simple
text.

~~This can be tested in combination with https://github.com/nextcloud/contacts/pull/2587~~
This can be tested with the *Contacts Interaction* app shipped in server. The second commit adds the Groupware team as co-maintainer and this now renders as expected on the apps management page.

### Before

Bildschirmfoto von 2021-12-27 12-26-46

### After

Bildschirmfoto von 2021-12-27 12-26-26

### CODE REVIEW

File: apps/settings/src/views/Apps.vue [hunk 1]
```diff
@@ -229,13 +229,19 @@ export default {
...
+			}
+
 			const author = Array.isArray(this.app.author)
-				? this.app.author[0]['@value']
-					? this.app.author.map(author => author['@value']).join(', ')
-					: this.app.author.join(', ')
-				: this.app.author['@value']
-					? this.app.author['@value']
-					: this.app.author
+				? this.app.author.map(authorName).join(', ')
```
Comment (tcitworld) [inline apps/settings/src/views/Apps.vue:243]: `Intl.ListFormat` would be nice, maybe another time.

**Review by tcitworld — APPROVED** [review]

File: apps/settings/src/views/Apps.vue (hunk 1 above)
Comment (ChristophWurst) [reply apps/settings/src/views/Apps.vue:243]: That is a nice helper. I also want to look into making authors with homepage/email clickable links. I'm opening individual PRs to keep changes separate that only go into 24+ and the fixes that we'll backport.

**Review by skjnldsv — APPROVED** [review]

**Pull request merged**